red_queen/games/applications/MQT/bench/benchmarks/qasm/ae.qasm

```diff
@@ -0,0 +1,25 @@
+OPENQASM 2.0;
```

Comment:
Do we need the qasm files anymore if we're going straight from mqt -> QuantumCircuit?